src/org/zaproxy/zap/extension/todo/ZapAddOn.xml

```diff
@@ -0,0 +1,18 @@
+<zapaddon>
+	<name>ToDo-List</name>
+	<version>3.0</version>
```

Comment:
There was restriction about version before 2.7. So I added this! Changing it right now :)